docs/source/admin/quick_howto/ciab.rst

```diff
@@ -310,3 +310,15 @@ Dante's socks proxy is an optional container that can be used to provide browser
 #. Once the CDN-in-a-box stack has started, use the aforementioned browser to access traffic portal via the socks proxy on the docker host.
 
 .. seealso:: `The official Docker Compose documentation CLI reference <https://docs.docker.com/compose/reference/>`_ for complete instructions on how to pass service definition files to the ``docker-compose`` executable.
+
+Static Subnet
+-------------
+Since docker-compose will randomly create a subnet and it has a chance to conflict with your network environment, using static subnet is a good choice.
```

Comment:
`docker-compose` is a command, should be wrapped in double-backticks. Or, if you're feeling like going above and beyond the call of duty, make it a manpage reference:

```rst
:manpage:`docker-compose(1)`
```